### DISCUSSION TITLE
Uncollapsed Leva Panel not resizing when number of inputs change

### DISCUSSION THREAD

**Discussion by alzatin** [pmndrs/leva#485] [Q&A]
Hello!
I currently have a couple of Leva Panels on my project and I'm trying to have one initialized uncollapsed. The panel initializes looking just fine, but since the inputs for the panel are changing dynamically with clicks on the screen, the size of the panel is failing to resize to accommodate the new number of inputs. I know dynamic resizing is possible because if I manually collapse and uncollapse the Panel it then begins resizing properly to the change of inputs. Any ideas on how I can fix it so it starts resizing properly from the start without having to uncollapse manually?

Starting with collapsed ={false}

<img width="319" alt="Screenshot 2024-01-02 at 11 51 12 AM" src="https://github.com/pmndrs/leva/assets/47953734/955db04a-357a-40ba-8e7c-05fac61b1fa8">

<img width="332" alt="Screenshot 2024-01-02 at 11 40 49 AM" src="https://github.com/pmndrs/leva/assets/47953734/25b4fdd9-b2a3-4bb9-8095-d6dc58bf94b8">

After manually collapsing and uncollapsing:

<img width="331" alt="Screenshot 2024-01-02 at 11 42 14 AM" src="https://github.com/pmndrs/leva/assets/47953734/410cce0d-0c33-4809-8ae1-4740779223c7">

**Comment by Remi-Tribia** [comment]
Hi,
I noticed it happens when using <Leva> component, when just using useControls() it seems to expand correctly.
